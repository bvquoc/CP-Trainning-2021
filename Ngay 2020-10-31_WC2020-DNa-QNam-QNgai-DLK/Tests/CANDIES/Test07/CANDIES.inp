100
126237172312832 1
964212771526736 2
851878578518194 3
214187599970163 4
411825553364287 5
814418447785372 6
611227512285513 7
559123881812575 8
563967351503259 9
119738688452785 10
401645111068399 11
418605197807181 12
988771968668452 13
552808502996980 14
379586855780831 15
699209765952724 16
764738340625800 17
646609981850312 18
815182511278085 19
279837389713402 20
525582662021782 21
278287801127093 22
372773113579146 23
107696527008423 24
462882773503888 25
104905724824333 26
674672994956900 27
170219782346807 28
471917383576763 29
499944045402594 30
762473599478605 31
981294936781640 32
896762552561816 33
817278666637646 34
977593112698323 35
525885878200580 36
715791867313985 37
474157504172137 38
904747632250191 39
384110363889044 40
989226833574045 41
505486617095996 42
405227774907262 43
673086703565490 44
284626768117284 45
302370060489212 46
621354029674975 47
322164917622783 48
159359266509735 49
453466339172588 50
961691424079617 51
803867943267285 52
133743200001611 53
956080244900075 54
528997895937734 55
534218489697111 56
786289836295427 57
532307392974491 58
363864222242063 59
641887845881709 60
776127486656884 61
975971006732052 62
448359252242965 63
728552333373073 64
152199377547208 65
764475738610638 66
909050184260612 67
646860950111010 68
654082720802904 69
889957578461641 70
944373796544527 71
701877054417172 72
855308157036809 73
721778813211127 74
893060528016755 75
358718814636817 76
813211071019971 77
842623905620529 78
740297350010966 79
762321788540345 80
961100650668837 81
426427236978186 19
738284150609843 34
293397063868646 69927008149115
677899021948882 29
294130913371801 26
191563300981115 33
347732221205245 47
150609434153587 411173536047023
887411967507958 17
484070743900103 26
867746455402792 15
424443086061802 23
781364224129159 21
747059781699636 10
379217762469993 514863073805285
985066437868041 9
731458670483498 19
237951550170171 45
450077380904896 15