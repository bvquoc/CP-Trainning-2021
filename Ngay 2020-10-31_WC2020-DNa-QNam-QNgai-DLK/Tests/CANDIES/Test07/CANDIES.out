2351786069480362
19064380362933421
16816740846399068
4052080216466469
7947417961990434
16092588182609791
11930375954881556
10885025303099368
10981628315571482
2232586676221949
7739825693940416
8081968010700579
19546432099550647
10692576609827052
7155500730884949
13193265800964936
13864330230323285
10511934870571249
11906812384077478
2632222215899493
4009891756785806
1045662527824246
808041619998295
47575280513090
203538498051518
3441666238477
33934052784011
309102331730
568060214216
103769483920
21368953664
2720813244
110434214
2533056
23416
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
5659296965168619
528534
0
1226301335257
28456810422008
236810
0
0
16153008662111142
73891510094177
16887093267309365
1008202547332103
6659681198579045
14691382560103926
0
19495111841343861
10442693441039336
0
8552505830194797
